### PR TITLE
Bug 1166595 - Blur the activity opener before opening activity window

### PR DESCRIPTION
**Pull request by alivedise**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=025f40eb5404387043ad49f76461ce618d2a6954

File: apps/system/js/browser_mixin.js [hunk 1]
```diff
@@ -73,6 +73,8 @@
       }
       this.debug(topWindow.name + ':' + topWindow.instanceID +
         ' is setting nfc active to: ' + enable);
+      console.log(topWindow.name + ':' + topWindow.instanceID +
+        ' is setting nfc active to: ' + enable);
```
Comment (etiennesegonzac) [inline apps/system/js/browser_mixin.js]: let's not forget to remove this one

File: apps/system/js/child_window_factory.js [hunk 2]
```diff
@@ -250,12 +250,15 @@
   ChildWindowFactory.prototype.createActivityWindow = function(evt) {
     var configuration = evt.detail;
     var top = this.app.getTopMostWindow();
+    console.log(top.name);
```
Comment (etiennesegonzac) [inline apps/system/js/child_window_factory.js]: and this one :)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5ae5f04115612228bbb6d9c80e90fa298fa328a4

**Pull request merged**